### PR TITLE
Add completable XBlock types installed in Tahoe to list of types affecting completion marks in course outline

### PR DESCRIPTION
**Pull request by bryanlandia**
Unfortunately, you have to update this list in openedx.core.features.course_experience.utils to add all completable types in order for them to have an effect on which parts of the outline are marked complete.

### TODO

 - [x] Omar to enable related tests to the `tox.ini` file
 - [x] ~~Bryan fix tests if needed~~
 - [ ] ~Omar to check if the latest LTI Consumer XBlock is compatible with Hawthorn~

### CODE REVIEW

**Comment by bryanlandia** [comment]
See https://appsembler.atlassian.net/browse/BLACK-1086 - Chef users are getting confused as to whether they've completed a course since Problem Builder blocks aren't showing up as complete in the outline, even though there is a green check mark when actually viewing the unit.

**Comment by OmarIthawi** [comment]
Thanks @bryanlandia I've added a checklist for this PR. I've added tests myself, let's see if it passes.

**Comment by bryanlandia** [comment]
Thanks for doing that Omar @OmarIthawi the tests passed

File: openedx/features/course_experience/utils.py [hunk 1]
```diff
@@ -133,7 +133,17 @@ def mark_last_accessed(user, course_key, block):
         'discussion',
         'drag-and-drop-v2',
         'poll',
-        'word_cloud'
+        'word_cloud',
+        'lti',
+        'lti_consumer',
```
Comment (OmarIthawi) [inline openedx/features/course_experience/utils.py:138]: not sure if this would work and I recommend removing it

Comment (bryanlandia) [reply openedx/features/course_experience/utils.py:138]: Am I missing something @OmarIthawi ? Not sure why latest LTI XBlock or not matters for this PR.  This is just a filtering list for what block types to request from the blocks API, to then check for completion.  If we don't include lti and lti_consumer here then scored LTI blocks will not figure in to how the completions are marked on the course outline.

Comment (OmarIthawi) [reply openedx/features/course_experience/utils.py:138]: @bryanlandia my bad. I thought you said somewhere in the lines that this PR requires the _latest_ LTI Consumer XBlock to be installed. I think we're good to merge this.

**Pull request merged**